lib/stoplight.rb

```diff
@@ -24,11 +24,16 @@ module Stoplight # rubocop:disable Style/Documentation
 
 require 'stoplight/default'
 
+require 'stoplight/configuration'
 require 'stoplight/light/lockable'
 require 'stoplight/light/runnable'
 require 'stoplight/light'
 
 # @see Stoplight::Light#initialize
```

Comment:
Since we now allow to return either Light or Configuration object from this method, I'd fix the documentation here.